### PR TITLE
Nav bar section

### PR DESCRIPTION
**Pull request by Sharonsyra**
**What does this PR do?** 
- It adds a new look to the navigation bar
- It improves the image size
- It fixes the bug where there was no menu icon on a small screen
- Partners' section fixes

**Description of the tasks**
- The links on the navigation bar on the second row should be moved to the top row.
- Use arrow_forward icons instead of arrow_down on the main section
- Use images with no background on the partners' slides
- Change button value depending on if one is logged in or not 
- Have the same color when one hovers on button
- Update links on button(s)
- It adds padding to the image to make it look bigger
- It shows the missing menu icon on a small screen
- It adjusts the size of the partners' section

**How to test it** 

[Link to course builder](https://codeforafrica-coursebuilder.appspot.com/)

**Screenshots**
screen shot 2017-09-25 at 3 58 03 pm
screen shot 2017-09-25 at 4 05 07 pm
screen shot 2017-09-22 at 6 12 30 pm

### CODE REVIEW

**Comment by chegejames** [comment]
@Sharonsyra https://gmkr.io/s/59c8a2df3042c29e3bd0408d/0

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 1]
```diff
@@ -186,15 +186,15 @@ <h2 class="section-label">Testimonials</h2>
       <template is="dom-if" if="[[!currentUser.loggedIn]]">
         <a href="[[currentUser.loginUrl]]">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Start Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="[[currentUser.loginUrl]]">Start Learning</a>
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra let's remove target="_blank" for this.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 2]
```diff
@@ -186,15 +186,15 @@ <h2 class="section-label">Testimonials</h2>
...
           </div>
         </a>
       </template>
 
       <template is="dom-if" if="[[currentUser.loggedIn]]">
         <a href="#!explorer">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Keep Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="#!explorer">Keep Learning</a>
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra let's remove target="_blank" for this.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 1 above)
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra Is there a reason why we are repeating the href from https://github.com/andela-angene/coursebuilder-core/blob/26175dba36e93d74695ea706f0abdd323b7f0004/coursebuilder/modules/explorer/_static/components/pages/homepage.html#L187

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra Is there a reason why we are repeating the href from https://github.com/andela-angene/coursebuilder-core/blob/26175dba36e93d74695ea706f0abdd323b7f0004/coursebuilder/modules/explorer/_static/components/pages/homepage.html#L195

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 1 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: will do

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: done

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Sorry, this was an oversight.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 1 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: fixed

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra it happens. "If debugging is the process of removing software bugs, then programming must be the process of putting them in." Lol!

File: frontend/index.html [hunk 3]
```diff
@@ -12,72 +12,74 @@
...
-            <!-- Dropdown Trigger -->
-            <li><a class="dropdown-button" href="#!" data-activates="dropdown1">More<i class="material-icons right">arrow_drop_down</i></a></li>
-          </ul>
-
-          <ul class="right hide-on-med-and-down sign-in-nav">
-            <li><a class="sign-in-link" href="#!" target="_blank">SIGN IN</a></li>
-            <li><a href="#!" target="_blank" class="waves-effect waves-light btn">Start Learning</a></li>
-          </ul>
-      </div>
+
```
Comment (chegejames) [inline frontend/index.html]: @Sharonsyra Did we just introduce a blank line that then triggered all these additions?

File: frontend/dashboard.html [hunk 4]
```diff
@@ -12,73 +12,76 @@
   </head>
 
   <body>
+
```
Comment (chegejames) [inline frontend/dashboard.html]: @Sharonsyra Same thing here? blank new line?

File: frontend/index.html (hunk 3 above)
Comment (Sharonsyra) [reply frontend/index.html]: I was just spacing to separate different components

**Pull request merged**